android/example/app.js

```diff
@@ -12,11 +12,6 @@ win.add(btn);
 win.open();
 
 btn.addEventListener('click', function(){
-
-	if (!TiIdentity.isSupported()) {
-		alert("Touch ID is not supported on this device!");
-		return;
-	}
```

Comment:
We don't want to remove this code since passcode might be disabled or if we're running on an old Android OS version. The `isSupported()` method needs to return `true` if device is capable of passcode support. See my previous comment for more.